src/app/courses-page/courses-page.component.html

```diff
@@ -1 +1,31 @@
-<p>courses-page works!</p>
+<div class="container">
+    <div class="row">
+        <div class="info">
+            <div class="info-header">
+                <div class="info-header-title">
+                    <h2>{{course.title}}</h2>
+                </div>
+
+                <div class="hours-minutes">
+                    <div class="timer">{{course.createHourAndMinutes()}}</div>
```

Comment:
please use custom pipes instead class methods for data modification within template